components/core/src/clp_s/search/Output.cpp

```
@@ -90,10 +97,12 @@ bool Output::filter() {
 
     std::string message;
     auto const archive_id = m_archive_reader->get_archive_id();
+    bool scanned_any_ert{false};
```

Cppcheck (2.20.0)

[style] The function 'get_count' is never used.

(unusedFunction)